desktop/app.js

```diff
@@ -74,6 +75,24 @@ module.exports = function main() {
       );
     });
 
+    ipcMain.on('clearCookies', function() {
+      session.defaultSession.cookies.get({}, (error, cookies) => {
+        cookies.forEach(cookie => {
+          let cookieUrl = '';
+          cookieUrl += cookie.secure ? 'https://' : 'http://';
+          cookieUrl += cookie.domain.charAt(0) === '.' ? 'www' : '';
+          cookieUrl += cookie.domain;
+          cookieUrl += cookie.path;
+
+          session.defaultSession.cookies.remove(
+            cookieUrl,
+            cookie.name,
+            () => {} // Ignore callback
+          );
+        });
+      });
+    });
```

Comment:
I think this one is going to need some explanation here - what we are doing, why we are building this URL, why we are looping through `cookies`, etc… can you provide some explanation or a link to an explanatory post?

Reply:
Added some comments in 92b1fde. Thanks @dmsnell!